metals/src/main/scala/scala/meta/internal/parsing/Trees.scala

```diff
@@ -174,7 +174,7 @@ final class Trees(
       val skipFistShebang =
         if (text.startsWith("#!")) text.replaceFirst("#!", "//") else text
       val input = Input.VirtualFile(path.toURI.toString(), skipFistShebang)
-      if (path.isAmmoniteScript || path.isMill) {
+      if (path.isAmmoniteScript) {
```

Comment:
Was this actually wrong? Do mill files have multiple stages like ammonite (don't remember why this was added actually) This actually caused `multi-stage` tests for ammonite fail, they might be wrong though.